package.json

```diff
@@ -60,6 +64,11 @@
     "react-addons-test-utils": "^0.14.2",
     "react-dom": "^0.14.2",
     "rimraf": "^2.4.2",
+    "rollup": "^0.21.0",
```

Comment:
This library originally used ES6 Module Transpiler, which changed to Esperanto, which is now Rollup. I'm all for the concept of module "folding" for building library bundles.